crates/bevy_window/src/cursor.rs

```diff
@@ -20,78 +20,130 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 )]
 #[reflect(Debug, PartialEq, Default)]
 pub enum CursorIcon {
-    /// The platform-dependent default cursor.
+    /// The platform-dependent default cursor. Often rendered as arrow.
     #[default]
     Default,
-    /// A simple crosshair.   
-    Crosshair,
-    /// A hand (often used to indicate links in web browsers).    
-    Hand,
-    /// An arrow. This is the default cursor on most systems.    
-    Arrow,
-    /// Indicates something is to be moved.    
-    Move,
-    /// Indicates text that may be selected or edited.    
-    Text,
-    /// Program busy indicator.    
-    Wait,
-    /// Help indicator (often rendered as a "?")    
+
```

Comment:
Not sure about all these empty lines.

Reply:
Copying the message in winit's 'keyboard.rs' verbatim to our 'keyboard.rs' file(Where `Key` and `KeyCode` are defined) will satisfy the W3C license :)

Reply:
from https://www.w3.org/copyright/software-license-2002/ ; I think we'd need to also include the changes made. they are pretty straightforward: 
- add Reflect capabilities
- use another feature name for serialization
- maybe not using non_exhaustive for enums?

But I lack knowledge on how to write that. I'll copy the licenses as I believe it's better than nothing :)